hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$wd/clean-deps.sh &&
```

Comment:
What are the `&&` accomplishing here? I can't remember off hand if they cause the newline to be ignored on parsing, but of what you want is conditional continue to next statements, use `set -o errexit` (which should be on anyway).